code/js/controllers/SpotifyController.js

```diff
@@ -5,25 +5,29 @@
       $ = require("jquery");
 
   var multiSelectors = {
-    playPause: ["#play-pause", "#play"],
-    playNext: ["#next", "#next"],
-    playPrev: ["#previous", "#previous"],
-    playState: ["#play-pause.playing", "#play.playing"],
-    iframe: ["#app-player", "#main"],
-    like: [".thumb.up", ".thumb.up"],
-    dislike: [".thumb.down", ".thumb.down"],
-    song: ["#track-name", ".caption .track"],
-    artist: ["#track-artist", ".caption .artist"]
+    play: ["#play-pause", "#play", ".buttons [title='Play']"],
+    pause: ["#play-pause", "#play", ".buttons [title='Pause']"],
+    playNext: ["#next", "#next", ".buttons [title='Next']"],
+    playPrev: ["#previous", "#previous", ".buttons [title='Previous']"],
+    playState: ["#play-pause.playing", "#play.playing", ""],
+    iframe: ["#app-player", "#main", ".top-container"],
+    buttonSwitch: ["false", "false", "true"],
+    like: [".thumb.up", ".thumb.up", ""],
+    dislike: [".thumb.down", ".thumb.down" ,""],
+    song: ["#track-name", ".caption .track", ".track-info-name"],
+    artist: ["#track-artist", ".caption .artist", ".track-info-artists"]
   };
 
   var controller = new BaseController({
     siteName: "Spotify",
-    playPause: "#play-pause",
+    play: "#play-pause",
```

Comment:
You can remove all of the properties in here except `siteName` since they are overwritten anyways in the loop over `multiSelectors` below.